src/index.html

```diff
@@ -10,23 +10,37 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="big-search"
+      data-qa="big"
+    >
+      <input
+        class="input input--big"
+        type="search"
+        pattern="\w+"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="small-search"
+      data-qa="small"
+    >
+      <input
+        class="input input--small"
```

Comment:
Same goes here
```
class="small-search__input input input--small"
```

Reply:
Also here, I really tried to use BEM syntax, but if I use it, brakes the nesting.